web/app/components/projects/add-to-or-create.hbs

```diff
@@ -0,0 +1,100 @@
+<Hds::Modal
+  data-test-add-to-or-create-project-modal
+  @onClose={{@onClose}}
+  class={{unless this.newProjectFormIsShown "search-window"}}
+  as |M|
+>
+  <M.Header data-test-modal-header>
+    {{if this.newProjectFormIsShown "Start a project" "Add doc to project"}}
+  </M.Header>
+  {{! TODO: confirm this height decision }}
```

Comment:
Did you mean to leave this TODO in? (fine if it will be addressed later, but wanted to make sure)

Reply:
nope, good catch